### PR TITLE
Hotfix/2626 next button

### PR DESCRIPTION
**Pull request by amdomanska**
1. Makes sure that <li> element in questions with radio buttons is focusable so that it can be focused instead of hidden input
2. fixes jquery selector for `nextBtn` in showTab

### CODE REVIEW

File: portality/forms/application_forms.py [hunk 1]
```diff
@@ -2719,9 +2719,9 @@ def __call__(self, field, **kwargs):
         html = ['<%s %s>' % (self.html_tag, html_params(**kwargs))]
         for subfield in field:
             if self.prefix_label:
-                html.append('<li>%s %s' % (subfield.label, subfield(**kwargs)))
+                html.append('<li tabindex=0>%s %s' % (subfield.label, subfield(**kwargs)))
```
Comment (amdomanska) [inline portality/forms/application_forms.py:2722]: @richard-jones I don't think this is the best place to put that but I cannot figure out what would better place be. IT does not feel like a new widget for formulaic either... What do you think?

Comment (richard-jones) [reply portality/forms/application_forms.py:2722]: I think you'll have to talk me through this one, I'm not sure what bug is being fixed, and what the trade-offs here are.

Comment (amdomanska) [reply portality/forms/application_forms.py:2722]: This line will fix https://github.com/DOAJ/doajPM/issues/2993

**Review by richard-jones — APPROVED** [review]
ok, I think this looks good, now I understand what the `tabindex` thing does here

**Pull request merged**